source/2019-08-15-octane-release-plan.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Octane is coming in v3.14
+author: Yehuda Katz, Jen Weber
+tags: 2019, Announcement, Recent Posts
+responsive: true
+---
+
+The Ember community is wrapping up the [Octane](https://emberjs.com/editions/) edition, and we expect to get it over the finish line over the next three releases!
+
+- The latest release of Ember is 3.12, which is a an LTS (long term support) candidate. As an LTS, it does not have any significant new features, but it does have many of Octane's features! Check out the [release blog posts](https://blog.emberjs.com) to learn about them.
```

Comment:
Is it important to state that the LTS has many Octane features? Saying it "does not have any significant new features" but also "it does have many of Octane's features" was a little confusing for me and seemed to imply that Octane itself doesn't have any significant new features.

Reply:
Simplified to:


- The next release of Ember is 3.12, which is a an LTS (long term support) candidate. Check out the [release blog posts](https://blog.emberjs.com) to learn which of Octane's features are already available in stable releases.